lib/rubocop/formatter/simple_text_formatter.rb

```diff
@@ -96,16 +104,24 @@ class Report
         include Colorizable
         include TextUtil
 
-        def initialize(file_count, offense_count, correction_count, rainbow)
+        def initialize(file_count, offense_count, correction_count, correctable_count, rainbow)
           @file_count = file_count
           @offense_count = offense_count
           @correction_count = correction_count
+          @correctable_count = correctable_count
           @rainbow = rainbow
         end
 
         def summary
           if @correction_count.positive?
-            "#{files} inspected, #{offenses} detected, #{corrections} corrected"
+            if @correctable_count.positive?
+              "#{files} inspected, #{offenses} detected, #{corrections} corrected,"\
+                " #{correctable} correctable"
```

Comment:
I'd probably use `auto-correctable` here, so it's clearer we're talking about offenses RuboCop can fix itself.